docs/com.bitwarden.contacts.md

```diff
@@ -0,0 +1,34 @@
+[Table of contents](README.md#table-of-contents)
+
+# Bitwarden contacts
+
+The `com.bitwarden.contacts` doctype is used to share
+[organizations](./com.bitwarden.organizations.md) with other users.
+
+A `com.bitwarden.contacts` may be linked to a [Cozy contacts](./io.cozy.contacts.md)
+by using the `email` attribute.
```

Comment:
How is materialized this link, is it a relationship in the `com.bitwarden.contacts.md`? Or a relationship in the `io.cozy.contacts`?

Reply:
Currently, no. But it is something that we should improve.